app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
I like `login.gov: Your confirmation code is XXX`.

Optimizing for lock screen visibility don't seem important to me.  If anything we might want to optimize for the opposite for security reasons.  But we can't really do either since lock screens vary.

I don't have super strong opinion here though so what ever you prefer.  However I would like to have differnt messages for confirming a phone and logging in using one.